ctl/lib/get_file.py

```diff
@@ -31,18 +32,34 @@ def __init__(self, config: Config):
             access_key=config.s3_access_key,
             secret_key=config.s3_secret_key,
             secure=False)
-    
-    def get_artifacts_dir_of(self, cluster_id: int) -> Object:
+        self._api_server_client = Minio(api_server_minio, secure=False)
```

Comment:
Seems `access_key` and `secret_key` are forgotten here.

BTW, I will prefer to extract them to another config entry (like `artifacts_s3_endpoint`, etc..), and fully ignore this config `s3_endpoint` here.

Reply:
It seems this minio address is fixed, and doesn't need access_key.